app/services/doc_auth/acuant/responses/get_results_response.rb

```diff
@@ -72,7 +72,7 @@ def create_response_info
             classification_info: classification_info,
             address_line2_present: !pii_from_doc[:address2].blank?,
             doc_auth_success: doc_auth_success?,
-            selfie_success: nil,
+            selfie_status: :not_processed,
```

Comment:
also since these are serialized as JSON in redis, this symbol will come back as a string

Reply:
@zachmargolis , yes we did discuss internally how this part works.

Reply:
Fixed!